docs/Serverless.md

```diff
@@ -0,0 +1,99 @@
+<h1 align="center">Serverless</h1>
+
+Run serverless applications and REST APIs using your existing Fastify application.
+
+> Fastify is not designed to run on the AWS Lambda platform.
+The Fastify framework is designed to make implementing a traditional HTTP/S server easy.
+The AWS Lambda platform services requests in a completely different manner than
+a traditional HTTP/S server and is therefore at odds with Fastify's whole design.
```

Comment:
Change this sentence to: `The AWS Lambda platform services requests differently than a standard HTTP/S server; thus, we cannot guarantee it will work as expected with Fastify.`

Reply:
✅